src/panels/config/devices/ha-config-device-page.ts

```diff
@@ -134,6 +136,8 @@ export class HaConfigDevicePage extends LitElement {
 
   @state() private _deviceAlerts?: DeviceAlert[];
 
+  private _deviceAlertsTimeout?: NodeJS.Timeout;
```

Comment:
```suggestion
  private _deviceAlertsTimeout?: number;
```

Reply:
```
Error: src/panels/config/devices/ha-config-device-page.ts(1166,7): error TS2322: Type 'Timeout' is not assignable to type 'number'.
```

Reply:
That means our TS settings are wrong, and it thinks it is run in Node.JS. But I think our tsconfig is correct at the moment...